### PR TITLE
Fix a regression introduced during egg smashing that broke the drmaa_library_path param

### PR DESCRIPTION
**Pull request by natefoo**
...to the drmaa runner (importing `drmaa` needs to be delayed until runner initialization so that the param to modify `$DRMAA_LIBRARY_PATH` can be evaluated).

### CODE REVIEW

File: lib/galaxy/jobs/runners/drmaa.py [hunk 1]
```diff
@@ -62,6 +54,14 @@ def __init__( self, app, nworkers, **kwargs ):
             log.info( 'Overriding DRMAA_LIBRARY_PATH due to runner plugin parameter: %s', self.runner_params.drmaa_library_path )
             os.environ['DRMAA_LIBRARY_PATH'] = self.runner_params.drmaa_library_path
```
Comment (nsoranzo) [inline lib/galaxy/jobs/runners/drmaa.py:56]: Maybe adding a comment here explaining why the import should be delayed may prevent the recurrence of this bug.

Comment (natefoo) [inline lib/galaxy/jobs/runners/drmaa.py:56]: Good idea! Done.

**Comment by nsoranzo** [comment]
:+1: Thanks @natefoo!

**Comment by dannon** [comment]
Globals, whee.  I can't think of a better fix though.

**Comment by natefoo** [comment]
@dannon That's the way it was before, so it's nothing new, but yeah, I was not fond of that either. Although since it's a module, it's possibly the most appropriate use of a global imaginable.

**Comment by natefoo** [comment]
Procedural question, do we remove status tags post-merge?

**Comment by dannon** [comment]
I think we should just leave them as-is, no need to change them.  When filtering, you can always search by is:open or is:merged to toggle whether or not they're included in results.

**Pull request merged**